specs/_features/eip7594/das-core.md

```diff
@@ -276,6 +281,10 @@ The potential benefits of having row custody could include:
 
 However, for simplicity, we don't assign row custody assignments to nodes in the current design.
 
-### Subnet stability
+### Why don't we rotate custody over time?
+
+To start with a simple, stable backbone, for now, we don't shuffle the custody assignments via the deterministic custody selection helper `get_custody_groups`. However, staggered rotation likely needs to happen on the order of the pruning period to ensure subnets can be utilized for recovery. For example, introducing an `epoch` argument allows the function to maintain stability over many epochs.
+
+### Does having a lot of column subnets make the network unstable?
 
-To start with a simple, stable backbone, for now, we don't shuffle the subnet assignments via the deterministic custody selection helper `get_custody_columns`. However, staggered rotation likely needs to happen on the order of the pruning period to ensure subnets can be utilized for recovery. For example, introducing an `epoch` argument allows the function to maintain stability over many epochs.
+No, the number of subnets don't really matter. What matters to the network stability is the number of nodes and the churn rate in the network.
```

Comment:
```suggestion
No, the number of subnets doesn't really matter. What matters to network stability is the number of nodes and the churn rate in the network.
```

Reply:
If we're going to have this paragraph at all I think it would be good to expand on it a bit, for example on the "what matters" part

Reply:
Agreed. I also thought it was a bit light on details.